src/components/Footer/FooterSocial/FooterSocial.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import FooterSocialItem from './FooterSocialItem';
+
+type FooterSocialProps = {
+  socialLinks: [];
+};
+
+export const FooterSocial = ({ socialLinks }: FooterSocialProps) => {
```

Comment:
Rename `socialLinks` props to `links` - "social" not needed as we're already inside the component.